packages/chart/src/components/LinearChart.jsx

```diff
@@ -209,13 +212,18 @@ export default function LinearChart() {
   }
 
   const countNumOfTicks = axis => {
+
+
+    
     // function get number of ticks based on bar type & users value
     const isHorizontal = config.orientation === 'horizontal'
     const { numTicks } = config.runtime[axis]
     let tickCount = undefined
 
     if (axis === 'yAxis') {
       tickCount = isHorizontal && !numTicks ? data.length : isHorizontal && numTicks ? numTicks : !isHorizontal && !numTicks ? undefined : !isHorizontal && numTicks && numTicks
+      // DEV 3163 - to fix edge case of small numbers with decimals
+      if (tickCount > max) tickCount = min < 0 ? Math.round(max) * 2 : Math.round(max); // cap it and round it its not an integer
```

Comment:
nit: readability

Reply:
It's just a ternary... not sure what you want here.
I can do this instead of all on one line... is this what you want?  
      if (tickCount > max) {
          // cap it and round it its not an integer
          tickCount = min < 0 ? Math.round(max) * 2 : Math.round(max); 
      }

Reply:
What you have is fine. I know it's a ternary. I was just pointing out that I think the cognitive load, at least for me, felt moderate. Maybe a small function or additional comments would help resolve that. It might be fine when you're actually in the code.